### ISSUE TITLE
Overwatch 2 is not launching after 2.13.0.1.131376 patch 

### ISSUE THREAD

**Issue by IVPXS** [italomandara/CXPatcher#217]
Personally had a Marqasa patch and it worked flawlessly right before patch 2.13.0.1.131376
Reinstalled Crossover several times, reapplied Marqasa patch several times
Then tried to apply CXPatcher to a freshly installed Crossover and freshly downloaded game - the same exact results.

Game launches with Battle.net and it shows itself in Activity Monitor, yet no game window shows up. Process itself uses some RAM, sometimes up to 13 gigs, but with no success to show a game window whatsoever.

Maybe this issue is possible to solve on a patch' end? This might be Blizzard decided to block VM's too, but it's above my grade to know for sure.

**Comment by ginotto8** [comment]
Yes! I am having the same exact problem! I've tried everything: reinstalling just crossover or the whole game, installing it from steam and trying again with the battle net launcher...

The game shows up in the activity monitor every time you click play but it just wouldn't launch. Also every time you press play it starts a new session in the background, bloating memory.

I have no idea on how to proceed..

**Comment by italomandara** [comment]
We'll try and find a solution, i need to download OW2 to test but if you please provide a crossover log that would be great

**Comment by ginotto8** [comment]
I would like to send the log file, but I think there are some private infos, can I send it to you in another way other than here?

**Comment by ginotto8** [comment]
While browsing I have also found [this](https://www.codeweavers.com/support/forums/general/?t=27;msg=312095).
So It may just be Overwatch's or Crossover's fault...

**Comment by IVPXS** [comment]
> you please provide a crossover log that would be great
Not sure how to provide it in the most correct way, but here I tried to launch Overwatch.exe with logging enabled

[Overwatch.cxlog.zip](https://github.com/user-attachments/files/17589725/Overwatch.cxlog.zip)

**Comment by Jalohann** [comment]
Would love to test and help find a solution. @italomandara

**Comment by italomandara** [comment]
It's a rosetta issue, we can't fix this and neither codeweavers
